CONVENTIONS.md

```diff
@@ -126,8 +126,8 @@ way.
 
 There are however three exceptions to this rule:
  - Use "world" for what Mojang calls "level" (see https://github.com/FabricMC/yarn/issues/89)
- - Use "container" for what Mojang calls "menu" (see https://github.com/FabricMC/yarn/issues/386)
- - Use "inventory" for what Mojang calls "container" (no issue yet, requires renaming "container" first)
+ - Use "screenHandler" or "handler" (depending on context; if the screen part is obvious it can be omitted) for what Mojang calls "menu" (see https://github.com/FabricMC/yarn/pull/1106)
```

Comment:
"screenHandler" -> "screen handler"

The exact capitalization (`screenHandler`, `ScreenHandler`, `SCREEN_HANDLER`) depends on the context. Or if you want to keep it screenHandler, at least replace the quotes with `code`.